README.md

```diff
@@ -130,6 +130,10 @@ You run an maintain your Orbs yourself. Your incidents are treated with a high p
 
 We run and maintain your Orbs on a supported provider of your choice. Your incidents are treated with a high priority. In return, we charge you with a price from CHF 500 per Orb and Month, decreasing to CHF 100.
 
+## Usage Data
+
+ORBOS components send errors and usage data to CAOS AG, so that we are able to identify code improvement potential. If you'd rather be anonymous or don't have an internet connection, pass the global flag `--disable-ingestion` when using orbctl. For disabling ingestion for already-running components, execute the takeoff command again with the `--disable-ingestion` flag.
+
```

Comment:
@mffap is this description enough?

Reply:
```suggestion
## Usage Data

ORBOS components send errors and usage data to CAOS AG, so that we are able to identify code improvement potential. If you don't want to send this data or don't have an internet connection, pass the global flag `--disable-ingestion` when using orbctl. For disabling ingestion for already-running components, execute the takeoff command again with the `--disable-ingestion` flag.

We try to distinguishing the environments from which events come from. As environment identifier, we defer the environment identifier from your git repository URL if the --gitops flag is passed.

Besides from errors that don't clearly come from misconfiguration or cli misuage, we send an inital event when any binary is started. This is a "<component> invoked" event along with the flags that are passed to it, except secret values of course.
```

Reply:
can we replace `CAOS AG` with `CAOS Ltd.`